CMakeLists.txt

```diff
@@ -43,6 +43,9 @@ enable_language(CXX C)
 # Enable C++11 support universally, with C++14 where available
```

Comment:
Yes, will do so.  It's fairly self-explanatory, and it's also in the cmake docs, so I could go with removal.